src/index.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import Calculator from './component/calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
 );
```

Comment:
- [ ] Great job and Everything working fine 💯 But according to the requirements you should import the `Calculator` component and use it in your **main component (App.js).** Not in the `index` directly 👍

So, Kindly add `App.js` file and import `Calculator` component into it, then inside `index.js` you can import `App.js` 

> I know that both ways are fine, But this is the convention in React 

![image](https://user-images.githubusercontent.com/45231276/198975170-fde81c96-e183-4657-871b-467a91073ec2.png)

> 💡 NOTE: Don't forget to make the `App.js` class based components